lib/logstash/outputs/s3.rb

```diff
@@ -130,6 +130,8 @@ class LogStash::Outputs::S3 < LogStash::Outputs::Base
   # Specifies wether or not to use S3's AES256 server side encryption. Defaults to false.
   config :server_side_encryption, :validate => :boolean, :default => false
 
+  config :storage_class, :validate => ["STANDARD", "REDUCED_REDUNDANCY", "STANDARD_IA"], :default => "STANDARD"
```

Comment:
@nevins-b Can you add documentation concerning this new option, maybe a high level summary and a link to the complete description at http://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html